### ISSUE TITLE
Using targetDir = 'components/' should not break packages

### ISSUE THREAD

**Issue by benjamincox** [yatskevich/grunt-bower-task#15]
I like using the grunt-bower-task, rather than just 'bower install', mostly because my CI server can simply run 'grunt' and not worry about whether it's a bower-savvy project.  However, I don't want it to install the optimized version of things into 'lib/' (or anywhere else) because my next Grunt step is requirejs, which does this for me.

My solution has been to set targetDir to 'components/'.  This makes it so the final copy step just puts the files where they already are.   

However, when installing backbone this causes the Grunt task to fail with:

```
Fatal error: ENOENT, no such file or directory 'components/backbone'
```

And, sure enough, there is no 'components/backbone' folder.   (Oddly, with any other value for targetDir there _is_ a 'components/backbone' folder present when the copy step begins.

Please allow us to either:
- use 'components/' as the targetDir
- bypass the final copy step altogether (maybe something like copyFiles = false)

**Comment by Globegitter** [comment]
Any update on this? Running into the same error and just want to leave everything in components. Would be brilliant if that would work.

**Comment by yatskevich** [comment]
Fixed in version 0.2.1
